src/shapes/object.class.js

```diff
@@ -1711,30 +1711,49 @@
       }
 
       var el = fabric.util.createCanvasElement(),
-          // skip canvas zoom and calculate with setCoords now.
           boundingRect = this.getBoundingRect(true, true),
-          shadow = this.shadow, scaling,
-          shadowOffset = { x: 0, y: 0 }, shadowBlur,
-          width, height;
+          shadow = this.shadow;
 
       if (shadow) {
-        shadowBlur = shadow.blur;
-        if (shadow.nonScaling) {
-          scaling = { scaleX: 1, scaleY: 1 };
```

Comment:
In the new code we are missing both the nonScaling attribute effect from the shadow and the getObjectScaling usage that will count for the groups